lib/rules/wrap-iife.js

```diff
@@ -46,64 +58,95 @@ module.exports = {
                 nextToken && nextToken.value === ")";
         }
 
-        return {
+        /**
+        * Get the function node from an IIFE
+        * @param {ASTNode} node to evaluate
```

Comment:
Looking at the other functions in this file, it looks like we use `* @param {ASTNode} node node to evaluate` rather than `* @param {ASTNode} node to evaluate`.